pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-format.md

```diff
@@ -198,7 +197,7 @@ Requests for images in the [`<amp-carousel>`](https://www.ampproject.org/docs/re
 
 ## Examples
 
-This section provides some AMPHTML Email code examples.
+This section provides some AMP for Email code examples.
```

Comment:
suggestion: add a link to the sample section `/documentation/examples.html?format=email`

Reply:
As above: `{{g.doc('/content/amp-dev/documentation/examples/index.html', locale=doc.locale).url.path}}?format=email` 🙂